### ISSUE TITLE
validation_data documentation entry incomplete

### ISSUE THREAD

**Issue by WurmD** [tensorflow/tensorflow#38950]
## Description of issue (what needs changing):
validation_data: Data on which to evaluate the loss and any model metrics at the end of each epoch. The model will not be trained on this data. validation_data will override validation_split. validation_data could be:

This is incomplete. It can also be 
A generator or keras.utils.Sequence returning (inputs, targets) or (inputs, targets, sample weights). A more detailed description of unpacking behavior for iterator types (Dataset, generator, Sequence) is given below.

### Correct links

I'm failing to find the respective location in https://github.com/tensorflow/docs of this page to make a pull request directly

**Comment by rohanreddych** [comment]
Hey @WurmD if you want to change the documentation then you need to make a pull request by changing the docstrings of the source file, the file for the above issue is https://github.com/tensorflow/tensorflow/blob/v2.1.0/tensorflow/python/keras/engine/training.py#L81-L2865 change what you think should be there and make a pull request.

**Comment by ymodak** [comment]
The documentation has been updated. Thanks!

**Comment by WurmD** [comment]
was it? the pull request wasn't merged https://github.com/tensorflow/tensorflow/pull/38979/files

**Comment by ymodak** [comment]
You are right. My bad.

**Comment by ymodak** [comment]
The documentation for `model.fit` arguments  are now updated with latest [tf-nightly api docs](https://www.tensorflow.org/api_docs/python/tf/keras/Model?version=nightly#fit). Thanks!
